.github/workflows/build.yaml

```diff
@@ -20,8 +20,10 @@ jobs:
         uses: actions/checkout@v3
       - name: normalize branch name for tagging
         run: |
-          NORM_TAG=$(echo "$GITHUB_REF_NAME" | tr -s "/" "-")
-          echo "NORM_TAG=$NORM_TAG" >> $GITHUB_ENV
+          BRANCH=$GITHUB_REF_NAME
+          TAG="${BRANCH////-}"
+          echo "TAG=$TAG" >> $GITHUB_ENV
+
```

Comment:
Why this change? The current version is already doing this but with a more readable syntax